Bottom: 56713c45e7181516168044a4f54083f4f577dd04
Top:    a5f2b6db71f4bf043b1a2a90b4a9df3aa7b357e5
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-12 19:16:53 +0000

Refresh of convert-layer-and-group

---

diff --git a/src/search.c b/src/search.c
index 6c00673..5e33b3d 100644
--- a/src/search.c
+++ b/src/search.c
@@ -1150,7 +1150,7 @@ SearchObjectByLocation (unsigned Type,
 {
   void *r1, *r2, *r3;
   void **pr1 = &r1, **pr2 = &r2, **pr3 = &r3;
-  int i;
+  Cardinal i;
   double HigherBound = 0;
   int HigherAvail = NO_TYPE;
   int locked = Type & LOCKED_TYPE;
